src/code-templates/services/order-service/jest.config.js

```diff
@@ -99,7 +99,20 @@ module.exports = {
   // projects: undefined,
 
   // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+  reporters: [
+    "default",
+    [
+      "@jest-performance-reporter/core",
+      {
+        "errorAfterMs": 1000,
+        "warnAfterMs": 500,
+        "logLevel": "warn",
+        "maxItems": 5,
+        "jsonReportPath": "performance-reports/performance-report.json",
+        "csvReportPath": "performance-reports/performance-report.csv"
```

Comment:
Should we turn this on only if some environment variable exists (in run-time, not in the generation process), as this could lead to unwanted generated files as this would probably only matter when you have problems.

Also,, is 2 formats necessary or can we just use JSON?

Reply:
I agree we can use only json.

These files will be generated only if we run with the reporters that appear in the config. I think they are harmless when generated locally.

The issue becomes apparent when during CI/CD processes they will be generated.  Because we are speaking about a fairly small file (unless you thousands of slow tests), I don't know if this is really an issue.
Nonetheless, we can add another `test:deploy` script (or any other name suggestion you have) that uses `--reporters='default`

WDYT?